components/browser/awesomebar/src/test/java/mozilla/utils/coroutines.kt

```diff
@@ -1,35 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package mozilla.utils
-
-import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.asCoroutineDispatcher
-import kotlinx.coroutines.test.resetMain
-import kotlinx.coroutines.test.setMain
-import java.util.concurrent.Executors
-
-/**
- * Create single threaded dispatcher for test environment.
- */
-fun createTestCoroutinesDispatcher(): CoroutineDispatcher =
```

Comment:
As this is public API, I think we should follow [the deprecating guideline](https://mozac.org/contributing/deprecating). What do you think?

Reply:
Is this a public API? I thought it was just a testing helper in the awesomebar component.

Reply:
Oh, sorry my bad you are right!